content/tutorials/git_merge_conflict/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Handle Git conflicts via GitHub.Com"
+description: "How to handle git conflicts using the GitHub website"
+authors: [thierryo]
+date: 2024-04-17
+categories: ["version control"]
+tags: ["git", "version control"]
+---
+
+## Fix merge conflict with a pull request
+
+Sometimes a pull request warnings about a merge conflict.
+A merge conflict occurs when one or more lines were altered both in the base branch as in the issue branch.
```

Comment:
```suggestion
A merge conflict occurs when one or more lines were altered in the base branch so that a merge conflict arises in the feature branch.
```